Pcap++/CMakeLists.txt

```diff
@@ -93,6 +98,7 @@ target_link_libraries(
          Packet++
          $<$<BOOL:${PCAPPP_USE_PF_RING}>:PF_RING::PF_RING>
          $<$<BOOL:${PCAPPP_USE_DPDK}>:DPDK::DPDK>
+         $<$<BOOL:${PCAPPP_USE_XDP}>:bpf>
```

Comment:
It would be nice to have a find_package(bpf)

So in case bpf is not found it fails at the configuration step. 
Also this make explicit where we have to look for lib and headers and allo user to have a custom BPF library with -DBPF_ROOT=<path_to_custom_bpf>

Reply:
How should I do that?

I found a few examples of `FindLibbpf.cmake`, is this what you mean?

- https://github.com/iovisor/bcc/blob/master/cmake/FindLibBpf.cmake
- https://fossies.org/linux/nghttp2/cmake/FindLibbpf.cmake

Reply:
The one from fossies seems nice but I would export a bpf::bpf target

Reply:
@clementperon I updated the PR and added `FindBPF` which exports a `BPF::BPF` target which is used in Pcap++ CMakeLists.txt. Please let me know what you think